subprojects/gdk_rust/tests/integration.rs

```diff
@@ -317,18 +317,42 @@ fn subaccounts(is_liquid: bool) {
     let mnemonic = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about".to_string().into();
     new_session.login(&mnemonic, None).unwrap();
 
-    // Allow some time for the new session to catch up
-    std::thread::sleep(std::time::Duration::from_secs(2));
+    // Wait until all subaccounts have been recovered
+    let mut i = 120;
```

Comment:
nit: 500ms in a test doesn't matter, this would be clearer if it was 60, the sleep was 1 sec, and the '1 minute' comment just said 'timeout out waiting for updates' since the delay time may be tweaked in the future.

Reply:
Updated in 69cd5b47d426b17e482a402f250c09499107ec61.

On a dedicated machine (which only builds and runs) it increases the test time by ~5s (up to 99s); but that's negligible since all integration tests are not run that often, and on busy boxes it's reasonable to expect that it will be even more negligible.